forge-gui/res/cardsfolder/upcoming/nalfeshnee.txt

```diff
@@ -0,0 +1,8 @@
+Name:Nalfeshnee
+ManaCost:5 R
+Types:Creature Beast Demon
+PT:4/6
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.wasCastFromExile | ValidActivatingPlayer$ You | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a spell from exile, copy it. You may choose new targets for the copy. If it's a permanent spell, the copy gains haste and "At the beginning of the end step, sacrifice this permanent." (A copy of a permanent spell becomes a token.)
+SVar:TrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | Amount$ 1 | MayChooseTarget$ True | UnearthEffect $ True
```

Comment:
does this work with `$` dangling like that?

also, is it bad that this doesn't check the "If it's a permanent spell" clause? @Hanmac @tool4ever dunno if there's some wacky corner case

Reply:
the dangling $ should be fixed, for the cause, i need to check the UneathEffect it is using there

Reply:
> the dangling $ should be fixed, for the cause, i need to check the UneathEffect it is using there

UnearthEffect is new in this PR also

Reply:
It checks internally here
```
                    if (sa.hasParam("UnearthEffect")) {
                        if (copy.getCardState().getType().isPermanent()) {
                            String unearthEffect = "DB$ Animate | Keywords$ Haste | Duration$ Permanent | AtEOT$ Sacrifice | Defined$ Self";
                            AbilitySub effectSub = (AbilitySub) AbilityFactory.getAbility(unearthEffect, copy.getHostCard());
                            effectSub.setActivatingPlayer(sa.getActivatingPlayer());
                            copy.setSubAbility(effectSub);
                        }
                    }
```